zipkin2/pom.xml

```diff
@@ -60,6 +60,12 @@
       <version>2.8.2</version>
     </dependency>
 
```

Comment:
heh no way jose ;) gave can't be a core dep as it conflicts

Reply:
I only need it for `TokenType` to determine v1 vs. v2. If zipkin2.Span is that only thing that is supported in the next week or two, then I can remove this. Otherwise, I need suggestions :).